### DISCUSSION TITLE
Git básico explicado con dibujitos

### DISCUSSION THREAD

**Discussion by RIvanCF** [Launch-X-Latam/MisionFrontEnd#285] [General]
## Git en dibujitos 
En lo personal, una de las mejores formas de estudiar y entender el git, es a través de dibujos y mapas (para las bifurcaciones en sus versificaciones de cualquier proyecto. 
Aquí les traigo unos dibujos que me ayudaron a entender lo mas básico y espero les ayude a mis compañeros que aun se les complica. 
Saluden a mi mascotita personal GitC4tB0t
Les recomiendo que sigan estudiando gente bonita y busquen sus mejores estrategias de estudio. 
![IMG_20221031_093203](https://user-images.githubusercontent.com/99302791/199049234-f30f1d62-e268-452c-9557-3a342c00d070.jpg)
![IMG_20221031_093211](https://user-images.githubusercontent.com/99302791/199049250-a76e4c5b-5bcc-4516-8208-be09bf22855f.jpg)

**Comment by AlvaroFierro** [comment]
Buen aporte 👍
